docs/data/base/components/table-pagination/table-pagination.md

```diff
@@ -1,39 +1,43 @@
 ---
 product: base
```

Comment:
Not related to this PR, but what is the purpose of `product` here ?
Do we need to add it on our X doc pages ?
And if so, should it be the same for the datagrid and the date pickers or two distincts ?

Reply:
Great question! Maybe @siriwatknp or @danilo-leal could explain?